crates/noirc_driver/src/main.rs

```diff
@@ -1,24 +1,32 @@
+use std::{env::current_dir, path::Path};
+
 use acvm::Language;
+use fm::FileManager;
 use noirc_driver::{
     add_dep, compile_main, create_local_crate, create_non_local_crate, CompileOptions,
 };
 use noirc_frontend::{
-    graph::{CrateType, LOCAL_CRATE},
+    graph::{CrateGraph, CrateType, LOCAL_CRATE},
     hir::Context,
 };
 fn main() {
     const EXTERNAL_DIR: &str = "dep_b/lib.nr";
     const EXTERNAL_DIR2: &str = "dep_a/lib.nr";
-    const ROOT_DIR_MAIN: &str = "example_real_project/main.nr";
+    const ROOT_DIR_MAIN: &str = "main.nr";
 
-    let mut context = Context::default();
+    let root = current_dir().unwrap().join("example_real_project");
+    let fm = FileManager::new(&root);
+    let graph = CrateGraph::default();
+    let mut context = Context::new(fm, graph);
```

Comment:
Thoughts on adding a struct method which would allow replacing these lines with `Context::new(&root)` to guarantee a fresh fm+graph?

Reply:
The FileManager should be separate from the Context, so I want it to always be specified